db/seeds/users.rb

```diff
@@ -1,2 +1,188 @@
+User.create(first: 'Beans', last: 'Tiramisu' , email: 'BeansTiramisu@Tiramisu.com', user_name: 'BeansTiramisu', street: '65852 Main St.' , city: 'Miami', state: 'FL', zip: 66666, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Chupa', last: 'Tiramisu' , email: 'ChupaTiramisu@Tiramisu.com', user_name: 'ChupaTiramisu', street: '63489 Main St.' , city: 'Denver', state: 'CO', zip: 80202, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Chups', last: 'Tiramisu' , email: 'ChupsTiramisu@Tiramisu.com', user_name: 'ChupsTiramisu', street: '44165 Main St.' , city: 'Springfield', state: 'MO', zip: 57294, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Carrot', last: 'Tiramisu' , email: 'CarrotTiramisu@Tiramisu.com', user_name: 'CarrotTiramisu', street: '4521 Main St.' , city: 'Paris', state: 'TX', zip: 44444, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Cake', last: 'Tiramisu' , email: 'CakeTiramisu@Tiramisu.com', user_name: 'CakeTiramisu', street: '48205 Main St.' , city: 'Nashua', state: 'NH', zip: 13062, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Chupa', last: 'Tootsie' , email: 'ChupaTootsie@Tootsie.com', user_name: 'ChupaTootsie', street: '45831 Main St.' , city: 'Nashua', state: 'NH', zip: 13062, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Chups', last: 'Topping' , email: 'ChupsTopping@Topping.com', user_name: 'ChupsTopping', street: '18125 Main St.' , city: 'Miami', state: 'FL', zip: 66666, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Ice', last: 'Topping' , email: 'IceTopping@Topping.com', user_name: 'IceTopping', street: '24817 Main St.' , city: 'Topeka', state: 'KS', zip: 94625, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Cream', last: 'Wafer' , email: 'CreamWafer@Wafer.com', user_name: 'CreamWafer', street: '81904 Main St.' , city: 'Miami', state: 'FL', zip: 66666, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Marshmallow', last: 'Wafer' , email: 'MarshmallowWafer@Wafer.com', user_name: 'MarshmallowWafer', street: '95689 Main St.' , city: 'Ashbury', state: 'AR', zip: 27482, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Cheesecake', last: 'Wafer' , email: 'CheesecakeWafer@Wafer.com', user_name: 'CheesecakeWafer', street: '2995 Main St.' , city: 'Nashua', state: 'NH', zip: 13062, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Souffle', last: 'Wafer' , email: 'SouffleWafer@Wafer.com', user_name: 'SouffleWafer', street: '43008 Main St.' , city: 'Fairfax', state: 'VA', zip: 22033, password: 'password', password_confirmation: 'password', admin: false)
```

Comment:
Line is too long. [245/80]<br>Space found before comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.